flit/__init__.py

```diff
@@ -47,6 +47,10 @@ def main(argv=None):
         help="Prepare flit.ini for a new package"
     )
 
+    parser_register = subparsers.add_parser('register',
+            help="register a package on PyPI"
```

Comment:
I'd like this to specifically say "...without uploading any files", to hint that that's the only case where you need it.